earth_enterprise/docker/README.md

```diff
@@ -16,55 +16,53 @@ Add users that you want to have access to Docker to the `docker` group:
 # adduser muser docker
 ```
 
-### Cent OS 7
+### CentOS 7
 
 Install Docker as root:
 
 ```BASH
 yum install docker
-# In case your Docker deamon isn't running:
+
+# in case your Docker deamon isn't running:
 service docker start
 ```
 
 Now you can use Docker as the root user.
 
+## Building Images
 
-## Building the Images
-
-Before building Open GEE Docker images, make sure you have, at least, about
-8 GB of free disk space.
+Before building Open GEE Docker images, make sure you have at least 8 GB of
+free disk space.
 
 ### Image Configuration
 
-There are a number of option of how you can build and run Open GEE Docker
-images: 
+There are a number of options for building and running Open GEE Docker
+images:
 
-    * Platform to build on,
-    * Source code origin: from the main repository, or a fork, from the
-      master, or another branch, or from the current repository clone on your
-      disk;
-    * Port to forward HTTP connections to;
-    * Custom image to run, e.g., containing extra data like tutorial
-      resources, or custom built globes.
+* Build platform
+* Source code origin: from the main repository, a fork, another branch,
+    or from the current repository clone on your disk
+* Port to forward HTTP connections
+* Custom image to run, e.g. one that contains extra data such as tutorial
+    resources or custom built globes
 
 These options are controlled by environment variables that are passed to the
-build scripts.  Since, it may become inconvenient to pass more than a couple
-of values on the command line, you could put the variable definitions in a
-shell script file instead, and source it before each command you execute.
+build scripts. Since it may become inconvenient to pass more than a couple
```

Comment:
I think you should use `a couple of` here.  We're not talking about exactly two values.